### ISSUE TITLE
User field mapping to entry's author attribute integration fails

### ISSUE THREAD

**Issue by fredrik-sogaard** [verbb/formie#307]
**Description**
I can't get map a user field to an entry's author field to work.

**Steps to reproduce**
1. Add user and text field to a form.
2. Set up an entry integration.
3. Map user field to entry's author attribute
4. Map a text field to entry's title
4. Open form an select user with admin permissions.
5. Submit form.

Form fails with error `Element integration failed for submission “80”. Error: Call to a member function one() on string /Users/fredriksogaard/*/*/vendor/verbb/formie/src/integrations/elements/Entry.php:180`

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.10
- Craft version: 3.5.17.1
- Multi-site: No

**Additional context**
When mapping the same user field to an entry's user field, everything works as expected.

**Comment by fredrik-sogaard** [comment]
I can make it work by changing this:
https://github.com/verbb/formie/blob/0556acb75abb58d5e502ecaef939a8bba49696c3/src/integrations/elements/Entry.php#L177-L185

To this:
```
foreach ($attributeValues as $entryFieldHandle => $fieldValue) {
    if ($entryFieldHandle === 'author') {
        if (isset($fieldValue[0])) {
            $entry->authorId = $fieldValue[0];
        }
    } else {
        $entry->{$entryFieldHandle} = $fieldValue;
    }
}
```
But I'm not sure that messes with something else?

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.11](https://github.com/verbb/formie/releases/tag/1.3.11)
